draft-ietf-quic-manageability.md

```diff
@@ -1023,6 +1023,39 @@ not arrive at their destination. Admitting a few packets allows the QUIC
 endpoint to determine that the path accepts QUIC. Sudden drops afterwards will
 result in slow and costly timeouts before abandoning the connection.
 
+## Guiding Path MTU
+
+Some networks support 1500-byte packets, but can only do so by fragmenting at a
+lower layer before traversing a smaller MTU segment, and then reassembling.
+This is permissible even when the IP layer is IPv6 or IPv4 with the DF bit set,
+because it occurs below the IP layer. However, this process can add to compute
+and memory costs, leading to a bottleneck that limits network capacity. In such
+networks this generates a desire to influence a majority of senders to use
+smaller packets, so that the limited reassembly capacity is not exceeded.
+
+For TCP, MSS clamping (Section 3.2 of {{?RFC4459}}) is often used to change
+the sender's maximum TCP segment size, but QUIC requires a different approach.
+Section 14 of {{QUIC-TRANSPORT}} advises senders to probe larger sizes using
+Datagram Packetization Layer PMTU Discovery ({{?DPLPMTUD=RFC8899}}) or Path
+Maximum Transmission Unit Discovery (PMTUD: {{?RFC1191}} and {{?RFC8201}}).
+This mechanism will encourage senders to approach the maximum size, driving
+fragmentation that they may not be aware of.
+
+Networks can seek to avoid this by identifying probable QUIC packets
+({{sec-identifying}}) and taking action whenever packets exceed the desired
+size. For DPLPMTUD a sufficient action is to drop the packet. If it is
+genuinely QUIC, the packet should have been a probe and so its loss should not
+cause impact to the flow of data. It serves only to guide the discovery of an
+optimal Path MTU.
+
+DPLPMTUD can optionally make use of ICMP Packet Too Big messages (PTB, defined
+in {{Section 1.1 of DPLPMTUD}}). PMTUD requires such PTB messages in order to
+work. Since the network cannot know in advance which discovery method the QUIC
+endpoints are using, it should send a PTB packet in addition to dropping the
+oversized probe packet. The generated PTB packet should be compliant with the
+validation requirements of section 14.2.1 of {{QUIC-TRANSPORT}} otherwise it
```

Comment:
```suggestion
validation requirements of {{Section 14.2.1 of QUIC-TRANSPORT}} otherwise it
```

Reply:
Why does this format give me the error "Error: IDREF attribute target references an unknown ID "section_14.2.1_of_QUIC-TRANSPORT", at None"?

Reply:
Try updating kramdown-rfc2629; it's a recent feature.

Reply:
I was using version 1.3.35; updating to 1.3.37 didn't help. However in debugging I realised that it's a case sensitivity problem. When the feature was introduced in 1.3.21 the notes said "Limitations for now: Only recognizes Section/Sections".

This means you cannot write {{section 14.2.1 of RFC1149}} but must write {{Section 14.2.1 of RFC1149}}. I know that was exactly the change you suggested but the resulting I-D then has an incongruous capital S in the middle of the sentence: "validation requirements of Section 14.2.1 of ". The output format is a feature of xml2rfc so I guess I have to accept that it must always be written as Section rather than section.

Reply:
It's not Carsten being obstinately German, it's good writing apparently.  Section is always capitalized as a proper noun.  It might not being especially proper, but that's the convention I was taught.